doc/installation.md

```diff
@@ -1,9 +1,95 @@
 Installation
 ============
 
-```{todo}
-* explain installation for developers.
-* explain installation from conda-forge, assuming we release it there soon.
-* explain installation from PyPI, assuming we release it there soon.
-* only leave the very basics of installation in the README and refer here instead.
+The recommended way to install the svgdigitizer is to use your package manager,
+(e.g., `apt-get` on Debian or Ubuntu, `pacman` on Arch Linux, `brew` on macOS.)
+
+You can consult [repology](https://repology.org/project/python:svgdigitizer/packages)
+to see if the svgdigitizer is available for your package manager.
+
+Alternatively, the svgdigitizer can easily be installed by one of the following
```

Comment:
```suggestion
Alternatively, the svgdigitizer can be installed by one of the following
```

Reply:
We tend to avoid words such as "easily" in our publications since this always depends on the users' skills.